src/pages/instances/InstanceOverviewDeviceDetail.tsx

```diff
@@ -46,24 +44,21 @@ const InstanceOverviewDeviceDetail: FC<Props> = ({ device, project }) => {
 
   if (device.type === "gpu") {
     return (
-      <span title={device.pci} className="u-truncate">
-        {device.pci ? `pci: ${device.pci}` : `id: ${device.id}`}
-      </span>
+      <div title={device.pci} className="u-truncate">
```

Comment:
why change these from `span` to `div`? I think semantically `span` makes sense?

Reply:
I wanted to get rid of the explicit line break. As divs are block elements they appear in dedicated lines and we can remove it.

Reply:
I wanted to get rid of the explicit line break. As divs are block elements they appear in dedicated lines and we can remove it.

Reply:
We can always use the regular for it, I think.

Reply:
nit: yes agreed with the approach for the `proxy` device section, since there are two lines. Although here it's a single line for the `gpu` device id, so `span` makes sense as it is semantically correct no?

Reply:
ah right, will change this one :+1: